exporter/honeycombexporter/go.mod

```diff
@@ -4,12 +4,12 @@ go 1.14
 
 require (
 	github.com/census-instrumentation/opencensus-proto v0.2.1
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.1
```

Comment:
Based on failures we saw in [another PR](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/230#issuecomment-629353362) that made the same change this results in performance regression and is likely the reason of perf test failures in this PR (I did not verify that it is actually the reason).

Reply:
This appears to happen because of a change in the HC exporter library. I didn't apply this change, the `go` tool did.

Reply:
@jmacd I think we can force this dependency to be 1.3.5?